index-fr.2.html

```diff
@@ -0,0 +1,121 @@
+<!DOCTYPE html>
```

Comment:
Can you please delete this file? It's redundant since actual changes are done in index-fr.html.